ext/CthulhuCompilerExt.jl

```diff
@@ -0,0 +1,16 @@
+module CthulhuCompilerExt
+
+@static if VERSION ≥ v"1.12.0-DEV.1581"
+    using Compiler: Compiler as CC
+    using Compiler.IRShow: IRShow
+    using Cthulhu: Cthulhu
+
+    function __init__()
```

Comment:
I don't think this should happen automatically - instead it should probably switch on the base version of the compiler.

Reply:
I'm not sure what you meant by that. Does that mean the Compiler stdlib is only loaded when the `Base.Compiler` and the actual Compiler stdlib implementation are different? Basically, only if someone has made local changes to the Compiler implementation.